src/index.js

```diff
@@ -15,6 +15,15 @@ if (window.location.href.split(':').includes('file')) {
   window.location.href = loginUrl;
 } else if (process.env.REACT_APP_SENTRY_ENABLED === 'true') {
   Sentry.init({
+    ignoreErrors: [
+      // plugins/extensions
+      // SingleFile
+      `undefined is not an object (evaluating 'e.optionsAutoSave')`,
```

Comment:
Je ne sais pas si faut passer en regex au cas où ? When using strings, they’ll partially match the messages, so if you need to achieve an exact match, use RegExp patterns instead.

Reply:
Du coup, on cible juste e.optionsAutoSave ?

Reply:
ha genre /^undefined is not an object (evaluating 'e.optionsAutoSave')$/ ?

Reply:
yes j'ai l'impression d'après leur doc